index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="./css/MobileStyle.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <!--Hero Section Start-->
+
+  <section id="hero">
+    <header>
+      <div class="hamburger">
+        <a href="#"><i class="fa fa-bars"></i></a>
+      </div>
+
```

Comment:
- [ ] Good job designing the first page of your portfolio 👏 , you should make sure that the `color` of the hamburger button stick to color given in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&mode=design) 

Your design
![Capture](https://github.com/Al-Amin-Khan-Shakil/Demo-Portfolio/assets/80458199/18865328-9825-42f6-b292-e23335dc9232)

Figma design

![Capture1](https://github.com/Al-Amin-Khan-Shakil/Demo-Portfolio/assets/80458199/ef3481f4-a414-4c5d-a0f3-412420ef5931)